packages/orbit-components/src/Accordion/components/AccordionWrapper/index.tsx

```diff
@@ -1,48 +1,15 @@
-import styled, { css } from "styled-components";
 import React from "react";
+import cx from "clsx";
 
 import type { Props } from "./types";
-import defaultTheme from "../../../defaultTheme";
-import type { Theme, ThemeProps } from "../../../defaultTheme";
-import mq from "../../../utils/mediaQuery";
 
-export const getBorder = ({ theme }: ThemeProps): string =>
-  `${theme.orbit.borderWidthCard} ${theme.orbit.borderStyleCard} ${theme.orbit.borderColorCard}`;
-
-export const getBorderRadius = ({ theme }: ThemeProps): string =>
-  `${theme.orbit.borderRadiusNormal}`;
-
-const CardElement = css`
-  ${({ theme, expanded }: { theme: Theme; expanded?: boolean }) => css`
-    width: 100%;
-    box-sizing: border-box;
-    position: relative;
-    box-shadow: ${expanded && theme.orbit.boxShadowActionActive};
-    border-top: ${!expanded && getBorder};
-    background: ${theme.orbit.backgroundCard};
-    ${mq.largeMobile(css`
-      border-left: ${!expanded && getBorder};
-      border-right: ${!expanded && getBorder};
-    `)};
-  `};
-`;
-
-const StyledAccordionWrapper = styled.div<{ expanded?: boolean }>`
-  ${CardElement};
-  border: ${getBorder};
-  border-radius: ${getBorderRadius};
-  transition: margin ${({ theme }) => theme.orbit.durationFast} ease-in-out;
-  margin: ${({ theme }) => theme.orbit.spaceXSmall} 0;
-`;
-
-StyledAccordionWrapper.defaultProps = {
-  theme: defaultTheme,
-};
-
-const AccordionWrapper = ({ dataTest, initialExpanded, ...props }: Props) => (
-  <StyledAccordionWrapper
+const AccordionWrapper = ({ dataTest, ...props }: Props) => (
```

Comment:
While reviewing this, I noticed that:
- This component is only used internally, in `AccordionSection`. It is not exported.
- No props are _ever_ passed to this component, except for `dataTest`.

With this in mind, I believe we can simplify this component. The `Props` can be simplified to just have `children` and `dataTest`. And the `expanded` condition is no longer needed, as I couldn't reproduce the addition of the said box-shadow.

Do you know any use case that contradicts what I said above?

Reply:
Great catch, thanks!

I’ve checked the history and this part of the component has been like this since its inception so safe to simplify 👍